src/app/components/user-job-card/user-job-card.component.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-job-item-container {
+  display: inline-block;
+  width: 180px;
+  margin: 5px;
+  background-color: $default-bg-color;
+  border-radius: $job-card-border-radius;
+  box-shadow: 0 2px 6px 0 rgba(0,0,0,0.4);
```

Comment:
Commas in function arguments should be followed by a single space

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceaftercomma)